FeedStoreChallenge/CoreData/Models/ManagedFeedImage.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Daniel Gallego Peralta on 17/8/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+internal class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: NSManagedObject
```

Comment:
`cache` property has type of `ManagedCache` in `FeedStore.xcdatamodel`. Please change its type here so that model and class declaration are consistent.

```suggestion
	@NSManaged var cache: ManagedCache
```